lib/templates_helpers/at_reCaptcha.js

```diff
@@ -0,0 +1,19 @@
+AT.prototype.atReCaptchaRendered = function() {
+    $.getScript('https://www.google.com/recaptcha/api.js');
```

Comment:
For the error you're getting: see [this SO question](http://stackoverflow.com/questions/27402217/googles-recaptcha-gives-uncaught-securityerror-after-successfull-validation-in) could it be worth a try?